lenny/configs/db.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import os
+import configparser
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+# Load config
```

Comment:
The config should only ever be loaded into the config variable by `/config/__init__.py`